src/webextension/widgets/panel.css

```diff
@@ -13,8 +13,9 @@
   display: flex;
   align-items: center;
   font-weight: 500;
-  height: 32px;
-  padding: 2px 5px;
+  background: #ededf0;
```

Comment:
This is a good change for consistency with the item list and the details, but there's one tiny thing left: in `src/webextension/list/popup/components/app.css`, you'll need to set the background color of the body to match this. This will ensure that the little arrow above the doorhanger is the right color.